src/components/Oscillators.js

```diff
@@ -34,10 +34,9 @@ export default function Oscillators({
 
   return (
     <div className="Function-Board">
-      <h2>SET OSCILLATOR</h2>
-
+      <p>SET OSCILLATOR</p>
       <div className="Vco-bar">
-        <h2>OSC 1</h2>
+        <p>OSC 1</p>
```

Comment:
Why do you changed this to paragraphs? headlines seems a good choice at first.

Reply:
As far as I remember, @marw didn't like the <h2>

Reply:
As far as I remember, @marwinburesch didn't like the h2 ...